pagopa/io-app
# chore(IT Wallet): [SIW-998] Add new element added toast notification in new wallet screen

> [!WARNING]
> This PR depends on https://github.com/pagopa/io-app/pull/5665, https://github.com/pagopa/io-app/pull/5648

## Short description
This PR adds the ability to show a toast when a new element/card is added to the wallet section.

## List of changes proposed in this pull request
- Added locale keys
- Added navigation parameters to routes when new metod/bonus is added to the wallet.

## How to test
With the `io-dev-api-server`, enable the **New wallet section** FF from the profile screen. Then, try to add a new bonus or payment method. You should be able to see a toast when returning back to the wallet section.

## Preview

https://github.com/pagopa/io-app/assets/6160324/c96c11ba-de63-4dd0-b968-5316c74c398d
